scanners/gitleaks/parser/parser.js

```diff
@@ -0,0 +1,45 @@
+HIGH_TAGS = ['JWT', 'Artifactory', 'AWS', 'PrivateKey'];
+MEDIUM_TAGS = ['Hash', 'Facebook', 'Twitter', 'Github', 'LinkedIn', 'Slack', 'Google', 'Heroku',
+  'Mailchimp', 'Mailgun', 'Paypal', 'Picatic', 'Teams', 'Jenkins', 'Stripe', 'Square', 'Twilio'];
+LOW_TAGS = ['Generic', 'FileName', 'FileExtension'];
```

Comment:
These should be defined using const or let.

```suggestion
const HIGH_TAGS = ['JWT', 'Artifactory', 'AWS', 'PrivateKey'];
const MEDIUM_TAGS = ['Hash', 'Facebook', 'Twitter', 'Github', 'LinkedIn', 'Slack', 'Google', 'Heroku',
  'Mailchimp', 'Mailgun', 'Paypal', 'Picatic', 'Teams', 'Jenkins', 'Stripe', 'Square', 'Twilio'];
const LOW_TAGS = ['Generic', 'FileName', 'FileExtension'];
```

Reply:
LOW_TAGS also seems unused